contrib/pom.xml

```diff
@@ -604,10 +604,10 @@
       <optional>true</optional>
     </dependency>
     <dependency>
-      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.5 -->
+      <!-- Kinesis depends on HttpPatch which requires httpclient 4.3.6 -->
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.3.5</version>
+      <version>4.3.6</version>
```

Comment:
I verified the KinesisInputOperator with httpclient = 4.3.6 and Kinesis dependency = 1.9.10. It's working fine.